src/Routes.jsx

```diff
@@ -98,7 +97,6 @@ const Routes = (props) => (
     <AuthenticatedRoute path="/admin_manage_lc/" component={AdminManageLC} props={props} rolesAllowed={[USER_ROLES.admin]} />
     <AuthenticatedRoute path="/admin_manage_dar_collections/" component={AdminManageDarCollections} props={props} rolesAllowed={[USER_ROLES.admin]} />
     {checkEnv(envGroups.NON_STAGING) && <AuthenticatedRoute path="/dataset_catalog/:variant" component={CustomDatasetCatalog} props={props} rolesAllowed={[USER_ROLES.researcher]}/>}
```

Comment:
Is this a place that also needs to be updated?

Reply:
This is the `custom dataset catalog functionality` in the PR description that I did not want to remove yet. Jonathan has shared out link to this so we need to figure out what to do with it.